components/net/fetch/methods.rs

```diff
@@ -359,27 +359,29 @@ pub async fn main_fetch(
             if (same_origin && request.response_tainting == ResponseTainting::Basic) ||
                 // request's current URL's scheme is "data"
                 current_scheme == "data" ||
+                // or is a fetchable custom protocol
```

Comment:
Continuing the specification sentence still makes this seem like part of the specification... How about this:

```suggestion
                // Note: Although it is not part of the specification we make an 
                // exception here for custom protocols that are explicitly marked 
                // as active for fetch.
```

Reply:
Sounds fine to me